### DISCUSSION TITLE
Please stop auto-closing documents.

### DISCUSSION THREAD

**Discussion by bdenison99** [sumatrapdfreader/sumatrapdf#3211] [General]
I use Sumatra PDF Reader to read some e-books on a PC.  I like it most of the time, but I have come across something that is really frustrating me.  I open a book, and am looking at a chapter on coding some challenge or whatever.  I leave Sumatra to go work in a source code editor, go to lunch, or whatever and come back and unlock the computer to find that Sumatra has closed the document I was working on.  Is there any way to turn this off?  I never, ever want to have any application close documents on me for any reason while the application is still running.  I don't care whether I've locked the screen or just stepped away for a while.  The application should leave the document open right where it is.

**Comment by GitHubRulesOK** [comment]
AFAIK There is no auto close. The same static local file should be open a week later. So the closure would be if the file is no longer accessible for  refresh or a key to close has been triggered by screen locking. I have just opened the PC from lastnights "sleep" and the same 2 PDF files are showing from yesterday.

**Comment by bdenison99** [comment]
In that case, this has to be something like Google-Drive or Microsoft's OneDrive causing the frustration.  I'd file the request with them, but it would likely be ignored.  Thank you!

**Reply by GitHubRulesOK** [reply]
sync drives will naturally mess with active files
SumatraPDF naturally needs local files
so open a cloud http://file.pdf file it will be localised as ALL browsers etc. do (make a local copy to discard)
However for synchronised local folders they need to be static.
Its why when working with Word or CAD file I learnt the hard way don't trust synchronisation , lost many man months of work several times.
